.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+
+### Description
+Please provide a clear and concise description of your changes. Explain **what** this pull request accomplishes and *
+*why** it is needed.
+
+---
+
+### Related Issue
+
+If this pull request addresses an existing issue, please link to it below. Use the `Fixes #<issue-number>` syntax to
+close the issue automatically when the pull request is merged.
+
+Example:
+
+- Fixes: #1234
+
+---
+
+### Contributor License Agreement (CLA)
+
+By contributing to this project, you agree to the terms of
+the [Canonical Contributor License Agreement (CLA)](https://ubuntu.com/legal/contributors).  
+If you have not already signed the CLA, please do so [here](https://ubuntu.com/legal/contributors).
+
+---
+
+### Commit Message for Squash Merge
+
+If this pull request is merged with the squash option, what should the commit message be?  
+Provide the desired commit message below:
+
+Example:
+[Category] Brief description of changes made
+
+---
+
+### Checklist
+
+- [ ] I have read and followed the [Ubuntu Server contributing guide](https://documentation.ubuntu.com/server/contributing/).
+- [ ] I have signed the [Contributor License Agreement (CLA)](https://ubuntu.com/legal/contributors).
+- [ ] My changes are well-documented, and I have updated the documentation as needed.
+- [ ] My pull request is linked to an existing issue (if applicable).
+- [ ] I have tested my changes, and they work as expected.
+- [ ] New and existing unit tests pass locally with my changes.
```

Comment:
Unit tests in contributions to this repository likely never have unit-tests as it is not code.
Some time in the future we most likely re-enable some spell checks and other linting and once we do so we'd add a better matching line here then. For now I'd suggest to remove it.